tests/playwright/src/specs/podman-machine-onboarding.spec.ts

```diff
@@ -128,6 +128,7 @@ test.describe.serial('Podman Machine verification', () => {
     test.skip(process.env.TEST_PODMAN_MACHINE !== 'true');
 
     test('Create a default Podman machine', async () => {
+      test.setTimeout(PODMAN_MACHINE_STARTUP_TIMEOUT + 30000);
```

Comment:
shouldn't the timeout be a constant itself ? like why do we add 30s to an existing constant ?

I would expect something to be always like `test.setTimeout(PODMAN_MACHINE_SOMETHING_TIMEOUT);` (and then there is a timeout for the creation of a machine) else if we increase/decrease timeout per test, I'm not sure we need to declare these constants anyway

Reply:
The constant is used to define the wait time for the podman machine creation, but the test itself does more than just create the machine so it need to be longer than the podman machine creation wait time.

Currently the podman machine creation time is however long the constant is initialized at but the test itself timesout at 60 seconds because that is the default value for playwright.

Reply:
I would expect to have like a `const PODMAN_FULL_STARTUP_TIMEOUT = PODMAN_MACHINE_STARTUP_TIMEOUT + 30000` (but where constants are defined, not in the test)

Reply:
agreed, changed to this form now.